Computer_Architecture/CA_3.10.5_Supporting_Variable-Size_Stack_Frames.md

```
@@ -0,0 +1,91 @@
+15	movq %rax, -8(%rbp)        # Store on stack
+.L2:
+17	movq -8(%rbp), %rax        # Retrieve i from stack
+18	cmpq %rdi, %rax            # Compare i:n
+19	jl .L3                     # If <, goto loop
+		# . . .
+	# Code for function exit
+20	leave                      # Restore %rbp and %rsp
+21	ret
+```
+
+![](https://i.imgur.com/NNUIKUE.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)